src/index.ts

```diff
@@ -181,6 +193,31 @@ export default class Position {
 		return weightedPlacements[0].placement;
 	}
 
+	findBestPlacementAt(
+		offset: Offset,
+		reference: Element,
```

Comment:
We can drop this to keep the base signature aligned with `findPlacementAt` :+1: The expectation right now is that consumers would provide their own `containerFunction` if needed ...